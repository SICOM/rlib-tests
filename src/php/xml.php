<?php
	$output_format = 'txt';
	if (isset($argv[1]))
		$output_format = $argv[1];

	$rlib =	rlib_init();
	rlib_version();
	rlib_set_output_format_from_text($rlib, $output_format);
	rlib_add_datasource_xml($rlib, "local_xml");
	rlib_add_query_as($rlib, "local_xml", "data.xml", "data");
	rlib_add_report($rlib, "array.xml");
	rlib_execute($rlib);
	header(rlib_get_content_type($rlib));
	rlib_spool($rlib);
	rlib_free($rlib);
?>
