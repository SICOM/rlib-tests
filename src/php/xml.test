COMMAND="php xml.php"
EXTRA_COMMAND=
FORMATS="pdf xml txt html"
EXPECTED_pdf="1 2"
EXPECTED_xml="1 2"
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html="1 2"
