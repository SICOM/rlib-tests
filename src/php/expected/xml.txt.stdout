  dude sweet  xxxxzzzzzz 12345
  dude sweet  xxxxzzzzzz 12345
  dude sweet  xxxxzzzzzz 12345
  dude sweet  xxxxzzzzzz 12345
  dude sweet  xxxxzzzzzz 12345
